recipe/meta.yaml

```diff
@@ -59,6 +59,7 @@ test:
     - pip
   commands:
     - pip check
+    - pytest --pyargs astropy -vv
```

Comment:
so are you running run_test.py as well as the full test suite...?

Reply:
I would assume so, but how I should verify that through the logs? And would it be best to remove this and/or move it to the run_test.py?

Reply:
yeah it looks like both are running. sorry, what I really meant was, are you meaning to run both? I think the intention in run_test.py is to just run a subset of tests, so this line should be removed.

Reply:
which will probably help with the linux-64 and win test failures...

Reply:
Okay. I'd honestly prefer to run the full suite and just move the imports from the `run_test.py` if possible. I've identified the failing test and it's the same on both platforms, but if that doesn't fix it, then I'll just remove as suggested and only run the subset already established in the run_test.py

Reply:
yeah I agree